Sponsor section updated

jchoi4524 opened the pull request:
More silver sponsors added. Gold sponsor is hidden due to temporary ART logo.

jchoi4524 committed:
Sponsor section updated

More silver sponsors added. Gold sponsor is hidden due to temporary ART logo.

jchoi4524 committed:
Merge branch 'main' into jc_preview

jchoi4524 committed:
recommiting

canadianyoyoassociation commented on faq.html:
```diff
@@ -80,14 +80,14 @@ <h3>Is there anything else related to music?</h3>
-        loadGoldSponsors("sponsor-footer", 0.6);
+        loadGoldSponsors("sponsor-footer", 0);
     </script>
     -->
```
Don't forget to delete the trailing comment marker

Same for all files

canadianyoyoassociation commented on index.html:
```diff
@@ -110,30 +110,31 @@ <h1>
     <script src="sponsors.js"></script>
     <script>
-        loadGoldSponsors("gold-sponsor-logos", 1);
+        loadGoldSponsors("gold-sponsor-logos", 0); <!--Scale set at 0 due to temporary ART logo-->
```
Better to just comment it out instead of setting scale to 0

canadianyoyoassociation commented on schedule.html:
```diff
@@ -84,15 +84,15 @@ <h1>Vendors</h1>
-        loadGoldSponsors("sponsor-footer", 0.6);
-		loadVendors("sponsor-logos", 1);
+        loadGoldSponsors("sponsor-footer", 0);
+		loadVendors("sponsor-logos", 0);
```
use spaces

jchoi4524 committed:
more refinement

1. replaced all taps with 4 spaces.
2. deleted redundant comment tails
3. commented out the portion of <script> in htmls to hide the gold sponsors for now.
